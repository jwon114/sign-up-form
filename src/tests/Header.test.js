import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react'
import Header from '../components/Header';

test('renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('has headings', () => {
  const { getByText } = render(<Header />);
  expect(getByText('Create a new account')).toBeInTheDocument();
  expect(getByText("It's quick and easy.")).toBeInTheDocument();
});

