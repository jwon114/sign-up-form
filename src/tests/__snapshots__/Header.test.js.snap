// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="Header"
>
  <h1
    className="Header__main"
  >
    Create a new account
  </h1>
  <h2
    className="Header__sub-heading mt-0 mb-m"
  >
    It's quick and easy.
  </h2>
</header>
`;
